spec/javascript/pages/Offer.spec.js

```diff
@@ -83,16 +84,5 @@ describe('Offer', () => {
         assert.equal($wrapper.get('input[name*="email"]').attributes('required'), 'required')
       })
     })
-
-    describe('when a contact field is updated', () => {
```

Comment:
is this spec not needed anymore?

Reply:
Yeah, it's not. Turns out i had over-complicated `ContactFields`.

This spec was testing that the parent object's value was updated by `ContactFields`, but:
> It actually doesn't need to emit update/change events up to its parent
because we don't need those updated values for anything. Those values
are stored in the corresponding inputs and submitted when the form
submits.